src/spatial/ArborX_LinearBVH.hpp

```diff
@@ -178,6 +151,18 @@ KOKKOS_FUNCTION
         typename Details::AccessValues<Values, PrimitivesTag>::memory_space,
         typename Details::AccessValues<Values, PrimitivesTag>::value_type>;
 
+template <typename ExecutionSpace, typename Values, typename IndexableGetter>
```

Comment:
Enable if ExecutionSpace is indeed an execution space {?)

Reply:
I see your point. I think it is out of scope for this PR. Rather, we would need to do that for all our deduction guides, both `BVH` and `BruteForce`. Or just wait for concepts.